benchmarks/generate_comparison.py

```diff
@@ -34,8 +34,8 @@ def _main():
         if os.path.isfile(path_csv):
             df_time = pd.read_csv(path_csv, index_col=0)
         else:
-            vanilla = vanilla_loop(cls_model, num_epochs=NUM_EPOCHS, num_runs=NUM_RUNS)
-            lightning = lightning_loop(cls_model, num_epochs=NUM_EPOCHS, num_runs=NUM_RUNS)
+            vanilla = measure_loops(cls_model, kind="Vanilla PT", num_epochs=NUM_EPOCHS, num_runs=NUM_RUNS)
```

Comment:
kind="Vanilla PT" -> use_lightning=False

Reply:
do we want eventually have some more in the future?

Reply:
Don't think so. If we do, we will refactor this part.

Reply:
ok, I'll refactor with the next PR which will add charts for larger models, the generation scrips need refactoring according to this update too...